api/tests/app/test_camera.py

```diff
@@ -0,0 +1,645 @@
+import pytest
+import os
+import base64
+import copy
+import numpy
+
+from api.tests.utils.common_functions import get_section_from_config_file, get_config_file_json
+
+# The line below is absolutely necessary. Fixtures are passed as arguments to test functions. That is why IDE could
+# not recognized them.
+from api.tests.utils.fixtures_tests import config_rollback, camera_sample, rollback_screenshot_camera_folder, h_inverse_matrix, pts_destination, rollback_homography_matrix_folder
+
+"""
+TO DO: Test stuffs related with the parameter reboot_processor.
+"""
+
+
+def create_a_camera(client, camera):
+    return client.post('/cameras', json=camera)
+
+
+def create_2_cameras(client, camera_base, enable=False):
```

Comment:
I think we should change this to use the `create_a_camera` method. Also, I dislike limiting it to two.

I'd recommend passing a number of cameras to be created and merging the previous PR

IDs should be handled by the previous merged feature, and the enable logic should be in `create_a_camera` 

The endpoint would return the list of cameras (maybe `create_a_camera`  should return  the params and the response)

Reply:
Thanks for pointing that out. I replaced it with a "create_n_cameras" function to solve this inconvenience.